db/projects/icon/project.yaml

```diff
@@ -0,0 +1,74 @@
+name: ICON
+alt_names:
+  - ICON Project
+  - Loopchain
+start_date: '2017-01-01'
+description: 'ICON provide that independent blockchains with different governances can transact with one another without intermediaries.'
```

Comment:
My idea:
description: 'ICON provides independent blockchains with different governances that could connect with each other without any intermediate party.'